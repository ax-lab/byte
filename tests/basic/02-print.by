print 'line 1'
print 'line 2'
print ''
print 'after empty line'
