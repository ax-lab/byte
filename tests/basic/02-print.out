line 1
line 2

after empty line
